nf_core/modules/modules_json.py

```diff
@@ -67,7 +67,8 @@ def create(self):
         new_modules_json = {"name": pipeline_name.strip("'"), "homePage": pipeline_url.strip("'"), "repos": {}}
 
         if not self.modules_dir.exists():
-            raise UserWarning("Can't find a ./modules directory. Is this a DSL2 pipeline?")
+            log.info(f"Creating ./modules directory in '{self.dir}'")
```

Comment:
It's no problem to add a prompt, but I don't know what DSL1 users are expecting, since I never used nf-core with DSL1. 

How about this?

`Can't find a ./modules directory. Would you like me to create one?` [y/n] (y)
if y -> `Creating ./modules directory in '{self.dir}'`
if n -> `UserWarning("Cannot proceed without ./modules directory.")`

Reply:
Perfect!